docs/setup_qa_pipeline.md

```diff
@@ -0,0 +1,42 @@
+# Steps to setup an end2end QA pipeline 
```

Comment:
Can you please add this directory structure to the docs as well? Some may clone the directories in different places.

Get all the repos in castorini and arrange them in this format
- castorini
    -- Anserini
    -- Castor
    -- data
    -- models